pkgs/data/icons/candy-icons/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenvNoCC
+, fetchzip
+, gtk3
+}:
+
+stdenvNoCC.mkDerivation {
+
+  pname = "candy-icons";
+  version = "0.6.8";
+
+  nativeBuildInputs = [ gtk3 ];
+
+  src = fetchzip {
+    url = "https://github.com/EliverLara/candy-icons/archive/37b4566b4bfcdad99131a236ef1fc74db89a302e.zip";
+    hash = "sha256-fgey2MduS9alUKiFhZr6KTKhYa68zh6LzEIcg/amrOo=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/icons
+    mv candy-icons-* $out/share/icons/candy-icons
+
+    for theme in $out/share/icons/*; do
+      gtk-update-icon-cache --force $theme
+    done
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "An icon theme colored with sweet gradients";
+    homepage = "https://github.com/EliverLara/candy-icons";
+    license = with licenses; [ gpl3 ];
+    maintainers = with maintainers; [ eneoli ];
```

Seems to be a missing `}` here? Also, let's set platforms as well!

```suggestion
    maintainers = with maintainers; [ eneoli ];
    platforms = platforms.all;
  };
```